ci/test-conformance-eks.sh

```diff
@@ -353,6 +366,119 @@ function cleanup_cluster() {
     echo "=== Cleanup cluster ${CLUSTER} succeeded ==="
 }
 
+function clusters_gc() {
+    echo "=== Starting EKS garbage collection in region ${REGION} ==="
+    echo "Cleaning up clusters older than ${GC_CLUSTER_AGE_HOURS} hours"
+
+    GC_CLUSTER_AGE_SECONDS=$((GC_CLUSTER_AGE_HOURS * 3600))
+    CURRENT_TIME=$(date +%s)
+
+    clusters=$(aws eks list-clusters --region ${REGION} --query 'clusters' --output json | jq -r '.[]')
+
+    if [ -z "$clusters" ]; then
+        echo "No EKS clusters found in region ${REGION}"
+        return 0
+    fi
+
+    deleted_count=0
+    skipped_count=0
+
+    for cluster in $clusters; do
+        # Skip clusters that don't match our naming pattern
+        if [[ ! "$cluster" =~ ^cloud-antrea-eks- ]]; then
+            echo "Skipping cluster $cluster (does not match naming pattern)"
+            skipped_count=$((skipped_count + 1))
+            continue
+        fi
+
+        creation_time=$(aws eks describe-cluster --region ${REGION} --name ${cluster} --query 'cluster.createdAt' --output text)
+
+        if [ -z "$creation_time" ]; then
+            echo "WARNING: Could not determine creation time for cluster $cluster"
+            continue
+        fi
+
+        creation_time=$(date -d "$creation_time" +%s)
+
+        if [ -z "$creation_epoch" ]; then
+            echo "WARNING: Could not parse creation time for cluster $cluster"
+            continue
```

Comment:
Maybe introduce a `failed_count` for such scenarios.

Reply:
Yes, I added `failed_count` to enhance the result.